dGame/dGameMessages/GameMessages.cpp

```diff
@@ -2468,6 +2431,8 @@ void GameMessages::HandleBBBSaveRequest(RakNet::BitStream* inStream, Entity* ent
 
 		Note, in the live client it'll still display the bricks going out as they're being used, but on relog/world change,
 		they reappear as we didn't take them.
+
+		TODO Apparently the bricks are supposed to be taken via MoveInventoryBatch?
```

Comment:
not necessarily move inv batch, but there's a BBB brick inventory. and the batch move is used a lot for when bricks are clone

Reply:
So this is only used for cloning?

Reply:
possibly? but not sure. It's definitely used for cloning and moving the brick that were cloned into the bricksInBBB inventory, but whatever bricks are in the inventory is what should be removed

move batch could be used for other things, just not sure off the top of my head

Reply:
Can be a separate PR with a config, but I just wanted to clear up the explanation of how it works since you left a comment about it and I didn't feel like I explained it before that well